appservice/src/main/java/nu/yona/server/AppServiceApplication.java

```diff
@@ -77,6 +77,30 @@ public void addCorsMappings(CorsRegistry registry)
 			// Enable CORS for the other resources, to allow testing the API through Swagger UI.
 			registry.addMapping("/**").allowedMethods("GET", "HEAD", "POST", "PUT", "DELETE");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "DELETE" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1192)